tests/infrastructure/infrastructure.go

```diff
@@ -26,5 +26,5 @@ import (
 )
 
 func DescribeInfra(text string, args ...interface{}) bool {
-	return Describe("[Serial][sig-compute]Infrastructure"+text, Serial, decorators.SigCompute, args)
+	return Describe("[Serial][sig-compute]Infrastructure "+text, Serial, decorators.SigCompute, args)
```

Comment:
It is out of the scope of this PR, but I really don't like hiding the `Serial` decorator behind a custom name.

Reply:
Agreed